tests/test_distributions.py

```diff
@@ -45,6 +56,16 @@ def test_json_to_distribution() -> None:
     pytest.raises(ValueError, lambda: distributions.json_to_distribution(unknown_json))
 
 
+def test_abbreviated_json_to_distribution() -> None:
+
+    for key in EXAMPLE_ABBREVIATED_JSONS:
+        distribution_actual = distributions.json_to_distribution(EXAMPLE_ABBREVIATED_JSONS[key])
+        assert distribution_actual == EXAMPLE_DISTRIBUTIONS[key]
+
+    unknown_json = '{"type": "UnknownDistribution", "low": 1.0, "high": 2.0}'
```

Comment:
[Nit] In this test,  a similar name to `int`, `float`, or `categorical` is supposed to be better than  
`*Distribution` since this test is for `type` not `name`. For example, 

```suggestion
    unknown_json = '{"type": "unknown", "low": 1.0, "high": 2.0}'
```